lewis/examples/epics_device/__init__.py

```diff
@@ -20,6 +20,7 @@
 from lewis.devices import Device
 
 from lewis.adapters.epics import EpicsInterface, PV
+from lewis.adapters.stream import StreamInterface, Var
```

Comment:
Seems a bit odd that the example called `epics_device` uses TCP stream... maybe rename it `dual_device` or something like that?